### PR TITLE
Added get_last_{week|month}_range and snap_to_beginning_of_{week|month].

### PR DESCRIPTION
**Pull request by Dexterp37**
As suggested by @mreid-moz , I'm adding these utility functions which are being used by the Fennec dashboard.

Where should I write test coverage (if that's needed in this case)?

### CODE REVIEW

**Comment by vitillo** [comment]
@Dexterp37 could you create a new testsuite for your functions with `nose`?

**Comment by Dexterp37** [comment]
I'm not sure how adding the tests affect the distribution of the moztelemetry library (e.g. are tests distributed? Should they be filtered out?).
Also, pardon any naive mistake using nose/mock, but that's my first time with them :)

File: README.md [hunk 1]
```diff
@@ -14,3 +14,8 @@ password:example_pass
 ```
 - Fetch the latest code with `git pull`
 - Update PyPI with `python setup.py sdist upload`
+
+## Running the tests
+
+- python setup.py install
```
Comment (vitillo) [inline README.md]: Installation shouldn't be required to run the tests.

File: README.md [hunk 2]
```diff
@@ -14,3 +14,8 @@ password:example_pass
...
+- python setup.py test
```
Comment (vitillo) [inline README.md]: It's preferable to invoke nose based tests with `nosetests` instead of with setuptools.

**Comment by vitillo** [comment]
Besides those two comments the rest looks good, thanks!

**Comment by Dexterp37** [comment]
I think this can be rebased to #63 when it lands.

**Comment by maurodoglio** [comment]
Yeah, the work on #63 is based on pytest instead of nose, but more or less everything else should work the same.

**Comment by maurodoglio** [comment]
@Dexterp37 #63 was merged, feel free to rebase your work on top of latest master.

File: moztelemetry/standards.py [hunk 3]
```diff
@@ -83,3 +83,46 @@ def mau(dataframe, target_day, past_days=28, future_days=10, date_format="%Y%m%d
     filtered = filter_date_range(dataframe, act_col, min_activity, max_activity,
         sub_col, min_submission, max_submission)
     return count_distinct_clientids(filtered)
+
+def snap_to_beginning_of_week(date, weekday_start="Sunday"):
```
Comment (mreid-moz) [inline moztelemetry/standards.py]: The `date` param name is the same as an imported module - please change to something else (I suggest `day`).

File: moztelemetry/standards.py [hunk 4]
```diff
@@ -83,3 +83,46 @@ def mau(dataframe, target_day, past_days=28, future_days=10, date_format="%Y%m%d
...
+    """ Get the first day of the current week.
+
+    :param date: The input date to snap.
+    :param weekday_start: Either "Monday" or "Sunday", indicating the first day of the week.
+    :returns: A date representing the first day of the current week.
+    """
+    delta_days = ((date.weekday() + 1) % 7) if weekday_start is "Sunday" else date.weekday()
+    return date - timedelta(days=delta_days)
+
+def snap_to_beginning_of_month(date):
```
Comment (mreid-moz) [inline moztelemetry/standards.py]: Same thing with the `date` param here

**Comment by mreid-moz** [comment]
Looks good!

**Pull request merged**